l10n_ar_wsfe/wsfetools/wsfe_easywsy.py

```diff
@@ -620,24 +616,22 @@ def validate_service_payment_date(val, invoice, Concepto):
     def validate_invoice_date(val, invoice, Concepto):
         if not val:
             return True
-        val_dt = datetime.strptime(val, AFIP_DATE_FORMAT)
-        val_odoo_format = val_dt.strftime(DATE_FORMAT)
+        val_date = datetime.strptime(val, AFIP_DATE_FORMAT)
         last_invoiced_date = invoice.get_last_date_invoice()
-        if last_invoiced_date and val_odoo_format < last_invoiced_date:
+        if last_invoiced_date and val_date.date() < last_invoiced_date:
             raise UserError(
                 _('WSFE Error!\n') +
                 _('There is another Invoice with a most recent date [%s] ' +
                   'for the same Point of Sale and Denomination.') %
                 last_invoiced_date)
         today = fields.Date.context_today(invoice)
-        today_dt = datetime.strptime(today, DATE_FORMAT)
-        offset = today_dt - val_dt
+        offset = today - val_date.date()
         if Concepto in [2, 3]:
             if abs(offset.days) > 10:
```

Comment:
it would be nice to have the 10 in a configuration so we don't hardcode it

Reply:
done